tools/run_tests/performance/dashboard_config/cloudbuild.yaml

```diff
@@ -0,0 +1,36 @@
+steps:
```

Comment:
what is this cloudbuild.yaml file for?

Reply:
I think the tools/run_tests/performance is not a good place to put this. 
It is where our legacy performance benchmarks live and we don't want to add more stuff there. I'm actually not convinced that the configuration belongs to the grpc/grpc repository at all

Reply:
This is the cloud build configuration for automated deployment